Users/Glob_Patterns.rst

```diff
@@ -123,14 +121,10 @@ brackets. Parentheses that have no match are ignored as well as
     False
```

Comment:
It justifies the statement that [!seq] here seq cannot be empty . If the seq is empty than no matter what pattern are you matching it with,  it'll always return false. unless the string and pattern are exactly same. i.e;
if I do
fnmatch("a[!]a","a[!]a")
It will return true. :)